arrow-libs/build.gradle

```diff
@@ -16,6 +16,11 @@ plugins {
     id "org.jlleitschuh.gradle.ktlint" version "$KTLINT_GRADLE_VERSION" apply false
     id "org.jetbrains.dokka" version "$DOKKA_VERSION" apply false
     id "ru.vyarus.animalsniffer" version "$ANIMALS_SNIFFER_VERSION" apply false
+    id "jacoco"
+}
+
+jacoco {
+    toolVersion = "$JACOCO_VERSION"
 }
```

Comment:
Please, add this piece in `arrow-libs/gradle/subproject.gradle` as well :raised_hands:

Reply:
In the end I removed it from here.. and move it all within `test-coverage.gradle`. May you check it out again please @rachelcarmena ? 🙏

Reply:
Good idea!! :clap:  I added another comment to unify the configuration :raised_hands: